### PR TITLE
Bundle Lua formatter into OSX bundle

### PR DESCRIPTION
**Pull request by vadi2**
This variant seems to include it correctly, though I haven't tested if it works at runtime - need https://github.com/Mudlet/installers/pull/35 merged for that first.

### CODE REVIEW

File: osx/make-installer.sh [hunk 1]
```diff
@@ -65,6 +65,8 @@ cp "${HOME}/.luarocks/lib/lua/5.1/rex_pcre.so" "${app}/Contents/MacOS"
 python macdeployqtfix.py "${app}/Contents/MacOS/rex_pcre.so" "/usr/local/opt/qt/bin"
 cp -r "${HOME}/.luarocks/lib/lua/5.1/luasql" "${app}/Contents/MacOS"
 cp "${HOME}/.luarocks/lib/lua/5.1/lua-utf8.so" "${app}/Contents/MacOS"
+cp -r "../3rdparty/lua_code_formatter" "${app}/Contents/MacOS"
```
Comment (keneanung) [inline osx/make-installer.sh]: If you use `cp -r "../3rdparty/lua_code_formatter/*" "${app}/Contents/MacOS/lfc"`, you should be able to leave out the mv

Comment (vadi2) [reply osx/make-installer.sh]: Done.

**Comment by vadi2** [comment]
[That broke it](https://travis-ci.org/Mudlet/Mudlet/jobs/310760462), so I had to revert it. Reverted versions works...

**Review by keneanung — APPROVED** [review]
Ok, lets keep it at that for the moment, though I'm not too happy with it.

**Comment by vadi2** [comment]
Sounds good - if you want to have a look at it later, we can revise.

**Pull request merged**